scripts/getDivviIntegrators.ts

```diff
@@ -136,11 +150,25 @@ async function getDivviIntegrators({
     usersThatHaveIntegrated,
   )
 
-  // TODO(ENG-345): Also filter for if the user is whitelisted
+  const fetchAllowlistResponse = await fetchWithTimeout(ALLOWLIST_URL)
```

Comment:
could we create a function that fetches this allow list (including the error handling / data massaging) and put this async function in the Promise.all above? i guess this api request will be fast, but just in case it's not?